src/Sulu/Bundle/ReferenceBundle/UserInterface/Command/RefreshCommand.php

```diff
@@ -24,6 +25,7 @@
  *
  * @final
  */
+#[AsCommand(name: 'sulu:reference:refresh')]
 class RefreshCommand extends Command
 {
     protected static $defaultName = 'sulu:reference:refresh';
```

Comment:
@alexander-schranz can we remove this line then?

Reply:
Not sure if its required for backwards compatibility. At current state when we remove it I would do it in own merge request in all commands.

Reply:
👍  makes sense